### ISSUE TITLE
Charts: scatter and box_whister

### ISSUE THREAD

**Issue by benoitgaudou** [gama-platform/gama-platform.github.io#94]
**Describe the bug**
I updated the documentation page about charts:
https://github.com/gama-platform/gama/wiki/DefiningCharts#box_whisker

Now I need help of charts experts:
* is the description of `scatter` correct? what is the meaning of the axis (in my simple example)
* I was not able to document `box_whister` as I was not able to make it work, and there is no example in the model library. Is it still useful : error box are available in another way if I am not wrong.

Another question:

For the scatter, I need to use `accumulate_values : true` in order to have several points. If I set it to `false`, I get only 1 point (whereas I was expecting to have a fix number of points but that change their locations over time...)

```
global {
    init {
	create people number: 10000;
    }
}

species people {
    float age <- gauss(40.0, 15.0);

    reflex older {
	age <- age + 1;
    }

    reflex die when: flip(age / 1000) {
	create people {
	    age <- 0.0;
	}

	do die;
    }
}

experiment my_experiment type: gui {
    output {
	display "my_display" {
	    chart "my_chart" type: scatter {
		data "Avg age" value: (people collect each.age) accumulate_values: true line_visible:false ;
	    }
	}		
    }
}

```

**Comment by pcaillou** [comment]
Hi Benoit!

To answer your questions:

- Scatter is exactly like xy. Both keyword exist because they already
existed in the previous version of the charts (and I never saw any
difference).

- Box and Whisker was never implemented... When I rewrote the charts,
nobody used it and it didn't work anymore. But since this is the second
time there is a question about it, I just started reimplementing it :) I
hope to finish soon!

Philippe

Le sam. 5 févr. 2022 à 05:53, benoitgaudou ***@***.***> a
écrit :
